documentation/SupportingDocuments/API_Documentation/Traffic_Influence_API_Documentation.md

```diff
@@ -218,8 +217,8 @@ Delete an existing TrafficInfluence resource
 | ---- | ---- | ----------- | -------- |
 | trafficInfluenceID | string | Identifier for the Traffic Influence resource. This parameter is returned by the API and must be used to update it (e.g., adding new users or deleting it). | No |
 | apiConsumerId | string | Unique Identifier of the TI API Consumer. | Yes |
-| applicationId | string | Unique ID representing the Edge Application<br>_Example:_ `"Virtual_Reality_Arena"` | Yes |
-| instanceId | string | Unique identifier generated by the partner OP to identify an instance of the application on a specific zone. | No |
+| AppId | string (uuid) | A globally unique identifier associated with the application. OP generates this identifier when the application is submitted over NBI.<br>_Example:_ `"6B29FC40-CA47-1067-B31D-00DD010662DA"` | No |
```

Comment:
Suggestion to change AppId, for appId for coherence with the rest of the table avoiding capital letters at the beginning, change proposed also for yaml.

Reply:
the parameters are defined in coherency with: https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/EdgeCloud_LcM.yaml

the parameters are **appId** and **appInstanceId,** while the schema name are **AppId** and **AppInstanceId.**

like:         
_appInstanceId:
$ref: '#/components/schemas/AppInstanceId'_

from line 598 of EdgeCloud-LcM

Reply:
It seems ok it most parts, schema is ok like that. It is just within TrafficInfluence Model, in the parameters included both in yaml ( line 357) and in Documentation file ( line 220), it seems more appropriate to keep consistency with the rest of parameters to start with lower case.